build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/InternalDistributionBwcSetupPlugin.java

```diff
@@ -120,6 +121,33 @@ private void configureBwcProject(Project project, BwcVersions.UnreleasedVersionI
             buildBwcTaskProvider,
             "assemble"
         );
+
+        // only run this section for versions where the stable projects would exist
+        if (bwcVersion.get().onOrAfter(Version.fromString("8.5.0"))) {
```

Comment:
Should this say 8.7.0, we haven't officially released the new stable API? Or maybe I'm wrong.

Reply:
You're right. I should have put in TODOs for that, as the older version numbers are useful for quick testing but need to be changed before merge.